components/dashboard/src/workspaces/ConnectToSSHModal.tsx

```diff
@@ -55,39 +55,46 @@ interface SSHProps {
 }
 
 function SSHView(props: SSHProps) {
-    const sshCommand = `ssh '${props.workspaceId}#${props.ownerToken}@${props.ideUrl.replace(
-        props.workspaceId,
-        props.workspaceId + ".ssh",
-    )}'`;
+    const [hasSSHKey, setHasSSHKey] = useState(false);
+
+    useEffect(() => {
+        getGitpodService()
+            .server.hasSSHPublicKey()
+            .then((d) => {
+                setHasSSHKey(d);
+            })
+            .catch(console.error);
+    }, []);
+
+    const host = props.ideUrl.replace(props.workspaceId, props.workspaceId + ".ssh");
+    const sshPswCommand = `ssh '${props.workspaceId}#${props.ownerToken}@${host}'`;
+    const sshKeyCommand = `ssh '${props.workspaceId}@${host}'`;
```

Comment:
We should make sure user who has ssh key uploaded can switch between `sshPswCommand` and `sshKeyCommand` since that private key may not exists in the computer user is using now. 

This need design too? @gtsiolis

Reply:
Hey @gtsiolis, do we need a design fot this? or it's good for us to use the similar style with image below from https://www.gitpod.io/ for switch?

|Switch| Button Style|
|-|-|
|![image](https://user-images.githubusercontent.com/20944364/174529632-ab33cdee-3545-49e0-9eac-334cfea97c1d.png) |![image](https://user-images.githubusercontent.com/20944364/174529759-1be5ee60-70db-4690-a36f-c5b5fcf762ee.png)|